### PR TITLE
SAM-2890 Styles lost and CKEditor not working when taking an assesment

### PR DESCRIPTION
**Pull request by alejandrogj**
using the "Published Assessment URL"

samigoassesurl3

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp [hunk 1]
```diff
@@ -62,7 +62,7 @@
       </style>
 
       <%@ include file="/jsf/delivery/deliveryjQuery.jsp" %>
-      <script type='text/javascript' src='/library/js/headscripts.js'></script><script type='text/javascript'>var sakai = sakai || {}; sakai.editor = sakai.editor || {};sakai.editor.enableResourceSearch = false;</script><script type='text/javascript'>var CKEDITOR_BASEPATH='/library/editor/ckeditor/';</script><script type='text/javascript' src='/library/editor/ckeditor/ckeditor.js'></script><script type='text/javascript' src='/library/editor/ckeditor.launch.js'></script>
+      <script type='text/javascript' src='/library/js/headscripts.js'></script><script type='text/javascript'>var sakai = sakai || {}; sakai.editor = sakai.editor || {};sakai.editor.enableResourceSearch = false;</script><script type='text/javascript'>var CKEDITOR_BASEPATH='/library/webjars/ckeditor/4.5.7/full/';</script><script type="text/javascript" src="/library/webjars/ckeditor/4.5.7/full/ckeditor.js?version="></script><script type="text/javascript" src="/library/editor/ckeditor.launch.js?version="></script>
```
Comment (juanjmerono) [inline samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp]: Could you use PortalService here to get the right CKEditor scripts?

**Comment by alejandrogj** [comment]
@juanjmerono I´ve changed the URLs of the Ckeditor for 
sakai-editor/editor-bootstrap.js
/sakai-editor/editor.js
/sakai-editor/editor-launch.js

**Comment by alejandrogj** [comment]
Please don´t merge this yet. I want to add the CSS class "Mrphs-sakai-samigo" to the jsp which renders the assesment to have all the styles available.

**Comment by alejandrogj** [comment]
Ready to be merged

File: samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp [hunk 2]
```diff
@@ -62,7 +62,11 @@
       </style>
 
       <%@ include file="/jsf/delivery/deliveryjQuery.jsp" %>
-      <script type='text/javascript' src='/library/js/headscripts.js'></script><script type='text/javascript'>var sakai = sakai || {}; sakai.editor = sakai.editor || {};sakai.editor.enableResourceSearch = false;</script><script type='text/javascript'>var CKEDITOR_BASEPATH='/library/editor/ckeditor/';</script><script type='text/javascript' src='/library/editor/ckeditor/ckeditor.js'></script><script type='text/javascript' src='/library/editor/ckeditor.launch.js'></script>
+      <script type='text/javascript' src='/library/js/headscripts.js'></script>
+      <script type='text/javascript'>var sakai = sakai || {}; sakai.editor = sakai.editor || {};sakai.editor.enableResourceSearch = false;</script>
```
Comment (juanjmerono) [inline samigo/samigo-app/src/webapp/jsf/delivery/deliverAssessment.jsp]: Please remove this line, is included in editor-bootstrap.js.

**Comment by alejandrogj** [comment]
Ok @juanjmerono i´ve removed that line

**Comment by juanjmerono** [comment]
Looks Good !!

**Comment by jonespm** [comment]
Interesting pattern, I'd forgotten about sakai-editor. I think I could use that in evaluation too. I was trying to figure out a good way to get access to the most recent version of the newest javascript and this pattern looks like half of the work!

**Pull request merged**